notes/01-templates.md

```diff
@@ -17,22 +17,21 @@ For now, paste the following HTML in [`app/templates/application.hbs`](../app/te
 ```hbs {starter-file=starter.html}
 <!-- Team Selector -->
 <nav
-  class="team-selector bg-indigo-darkest border-indigo-darkest border-r-2 pt-2 text-purple-lighter flex-none hidden sm:block">
+  class="team-selector bg-indigo-darkest border-indigo-darkest border-r-2 pt-2 text-purple-lighter flex-none hidden sm:block"
+  aria-label="Team navigation">
```

Comment:
this change needs to be added to `starter-files/index.html` as well

Reply:
Yes, it was added to starter-files/starter.html, as starter-files/index.html doesn't exist. :)